source/reference/mongosync.txt

```diff
@@ -182,6 +182,29 @@ show an empty ``system.views`` collection in that database. The empty
 ``system.views`` collection will not change the accuracy of user
 data on the destination.
 
+Write Blocking
+~~~~~~~~~~~~~~
+
+By default, write-blocking is not enabled during synchronization. When
+write-blocking is enabled, ``mongosync`` blocks writes:
+
+- On the destination cluster during replication
+- On the source cluster while committing
+
+To enable write-blocking, start ``mongosync`` with
+``enableUserWriteBlocking`` set to ``true``.
+
+:ref:`Reverse synchronization <c2c-api-reverse>` requires
+write-blocking. To enable reverse synchronization between the source
+and destination clusters, set ``reversible`` and
+``enableUserWriteBlocking`` to ``true`` when you use the
+:ref:`start API <c2c-api-start>` to initiate ``mongosync``. The options
+must be set at startup, they cannot be updated later.
```

Comment:
ditto, I still think `startup` is a bit unclear.

Reply:
Done